provider.go

```diff
@@ -6,67 +6,258 @@ import (
 	"net/url"
 	"strings"
 	"time"
+	"fmt"
+	"errors"
+	"os"
 
+	"github.com/coreos/fleet/log"
```

Comment:
It doesn't seem that any of the output generated with this ends up in the Terraform output. If we are to follow the convention established in the builtin providers, looks like we should just use the standard `log` module, indicating the severity with a flag. Example: https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/autoscaling_tags.go#L73